tools/code-analyzer/package.json

```diff
@@ -1,61 +1,34 @@
 {
 	"name": "code-analyzer",
-	"version": "0.0.0",
+	"version": "0.0.1",
 	"description": "A tool to analyze code changes in WooCommerce Monorepo.",
 	"author": "Automattic",
 	"bin": {
 		"code-analyzer": "./bin/run"
 	},
 	"homepage": "https://github.com/woocommerce/woocommerce",
 	"license": "GPLv2",
-	"main": "dist/index.js",
 	"repository": "woocommerce/woocommerce",
-	"files": [
-		"/bin",
-		"/dist",
-		"/npm-shrinkwrap.json",
-		"/oclif.manifest.json"
-	],
 	"dependencies": {
-		"@oclif/core": "^1",
```

Comment:
oclif is still required here for Code Analyzer to run

Reply:
@psealock it should no longer be required I did some follow up work to port all the commands away from oclif

Reply:
Looks like I left one oclif file behind! Fixed here https://github.com/woocommerce/woocommerce/pull/34600/commits/3a698360f3db76b148535526dd036bb8024434ff